questions/qa-bidi-unicode-controls.en.html

```diff
@@ -64,25 +64,32 @@ <h1>How to use Unicode controls for bidi text</h1>
+
+<p class="warning">If you are not familiar with how to manage bidirectional text in HTML markup, you may also find it useful to first read <a href="https://www.w3.org/International/articles/inline-bidi-markup/"><cite>Inline markup and bidirectional text in HTML</cite></a>.</p>
+
+
+
+
+
 <section id="reasons">
-<h2>Viable and non-viable use cases</h2>
-    <p>You will still need to use markup to establish the default direction for a document as a whole (eg. in the <code class="kw" translate="no">html</code> tag), and for block container elements. Because control codes don't cross paragraph (read as <span class="qterm">block element</span>) boundaries, and because control codes cannot manage inheritance and scoping through the markup hierarchy, they are only appropriate for inline use.</p>
-    <p><a href="/International/questions/qa-bidi-controls"><cite>Unicode controls vs. markup for bidi support</cite></a> explains that it is generally better to use markup, if  available, than to use control codes. Unicode control characters may, however, be necessary  in situations where markup is unavailable. Examples include legacy markup such as HTML elements that only contain plain text,  any HTML attribute value, and plain text formats such as WebVTT and CSV.</p>
-  </section>
+<h2>What this article does and doesn't cover</h2>
+    <p>You will still need to use markup to establish the default direction for a document as a whole (eg. in the <code class="kw" translate="no">html</code> tag), and to change direction for block container elements. Because control codes don't cross paragraph (read as <span class="qterm">block element</span>) boundaries, and because control codes cannot manage inheritance and scoping through the markup hierarchy, they are only appropriate for inline use.</p>
+<p>For example, although a <code class="kw" translate="no">title</code> element in the header of an HTML page can contain no markup, it is still possible to set a default base direction on the <code class="kw" translate="no">title</code> element itself (or for it to inherit the direction set on the <code class="kw" translate="no">html</code> tag). This article is about how to apply directional changes <em>within</em> the <code class="kw" translate="no">title</code> attribute text (or inline in other plain text situations), where needed, or how to apply a direction to attribute text which is different from that of the surrounding element, and so on. It also applies to plain text formats such as WebVTT and CSV.</p>
```

Comment:
Nit: I would change:

> ... although a `title` element in the header of an HTML page can contain no markup...

to read:

> ... although a `title` element in the header of an HTML page cannot contain markup...

(or possibly _additional markup_). The phrase "can contain no" just reads oddly to me.

Reply:
Fixed by https://github.com/w3c/i18n-drafts/pull/449/commits/639da7c9caaa113a9bf3d82b1121338134733efd